.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: tests
+
+on:
+  pull_request:
+    types: ['opened', 'synchronize']
+    paths-ignore:
+      - 'README.md'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'README.md'
+jobs:
+  unit:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      - 
+        name: Run tests
+        run: |
+          make test
+      - 
+        name: Install goveralls
+        env:
+          GO111MODULE: off
+        run: go get github.com/mattn/goveralls
+      - 
+        name: Send coverage
+        env:
+          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: goveralls -coverprofile=cover.out -service=github
```

Comment:
Nit: carriage return for the last line is missing.

Reply:
added :)